Player.java

```diff
@@ -1,37 +1,24 @@
-import java.util.*;
-import java.io.*;
-
-public class Player{
-   private String name;
-   private int space;
+public class Player
+{
+   private final String name;
+   private int currentSpace;

```

Comment:
original name was somewhat confusing.
actually... it should b called `currentPosition`...